spec/regenerate_swagger_docs

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+if [[ -z "${DISCOURSE_REPO_BASE_DIRECTORY}" ]]; then
```

Comment:
Let me know if there is a better env var for this, it is just the location of your source directory locally (e.g. mine is `~/repos`

Reply:
That should be the one to use. I have it set as well.